Userland/Libraries/LibCore/Stream.cpp

```diff
@@ -148,16 +148,29 @@ bool File::exists(StringView filename)
     return !Core::System::stat(filename).is_error();
 }
 
+ErrorOr<NonnullOwnPtr<File>> File::standard_input()
+{
+    return File::adopt_fd(STDIN_FILENO, OpenMode::Read);
+}
+ErrorOr<NonnullOwnPtr<File>> File::standard_output()
+{
+    return File::adopt_fd(STDOUT_FILENO, OpenMode::Write);
+}
+ErrorOr<NonnullOwnPtr<File>> File::standard_error()
+{
+    return File::adopt_fd(STDERR_FILENO, OpenMode::Write);
```

Comment:
These will explicitly ::close() stdin, stdout, and stderr when their File object is destroyed, which is explicitly different from old Core::File::standard_input() and friends.

Are we ok with these APIs being sublty different? I'm worried people will port more apps to use Core::Stream::File and not realize the differences b/w the two and introduce subtle bugs in our utilities.

Reply:
That could be an issue, but Core::Stream currently has no way of not closing the file when destroyed.
Should I add that as well then?

Reply:
Just added that